openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/DatabaseSchemaRepository.java

```diff
@@ -86,13 +86,14 @@ public void prepare(DatabaseSchema schema, boolean update) {
 
   @Override
   public void storeEntity(DatabaseSchema schema, boolean update) {
-    // Relationships and fields such as service are derived and not stored as part of json
+    // Relationships and fields such as service, database are derived and not stored as part of json
     EntityReference service = schema.getService();
-    schema.withService(null);
+    EntityReference database = schema.getDatabase();
+    schema.withService(null).withDatabase(null);
```

Comment:
Thanks for the quick PR @zjffdu . Could you please add the same thing for databaseSchema